docs/index.html

```diff
@@ -416,6 +416,11 @@ <h2>Structure</h2>
               <td>Mahmoud Alkhraishi</td>
               <td>mahmoud@mavennet.com</td>
             </tr>
+            <tr>
+              <td>Software Bill of Materials</td>
```

Comment:
Should we call this "software supply chain"? A software bill of materials is one of the possible certificates that can be issued.

Reply:
For sure!